application/src/main/resources/log4j2.xml

```diff
@@ -4,18 +4,27 @@
         <Console name="Console" target="SYSTEM_OUT">
             <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
         </Console>
-        <RollingFile name="File" fileName="logs/app.log" filePattern="logs/app-%d{dd-MM-yyyy}.log.gz"
+        <RollingFile name="File" fileName="logs/app.log"
+                     filePattern="logs/app-%d{dd-MM-yyyy}.log.gz"
                      ignoreExceptions="false">
-            <PatternLayout pattern="%d{yyy-MM-dd HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+            <PatternLayout charset="UTF-8"
+                           pattern="%d{yyy-MM-dd HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
             <Policies>
                 <SizeBasedTriggeringPolicy size="10 MB"/>
             </Policies>
         </RollingFile>
+        <Http name="Http" url="http://${env:TJ_LOG_TARGET:-localhost}:8080/rest/api/logs">
+            <JsonLayout/>
+        </Http>
+        <Async name="AsyncHttp">
+            <AppenderRef ref="Http"/>
+        </Async>
     </Appenders>
     <Loggers>
-        <Root level="info">
+        <Root level="Debug">
```

Comment:
Defaulf level should stay `Info`. To change the level we should provide config files and a way to change it hot while it runs, for example with this project (if possible) - but that might be part of another PR.

Reply:
Changed Back